runbotics/runbotics-orchestrator-ui/src/components/utils/Secured.tsx

```diff
@@ -23,6 +23,20 @@ export const hasAuthorities = (user: User, authorities: any[]) => {
     return true;
 };
 
+export const hasAdminAuthority = (user: User, roles: Role) => {
```

Comment:
naming does not match the purpose of the function, you need to rephrase it to be more generic

Reply:
see that you name parameter `roles` but type says that it is single one